src/Image/Image.js

```diff
@@ -37,12 +38,17 @@ const Image = ({
       className={st(classes.root, { tiled: isTiled }, className)}
       data-hook={dataHook}
       src={source}
-      style={{ ...style, ...imgStyle }}
+      style={{ ...style, ...imgStyle, ...{ borderRadius } }}
```

Comment:
I believe no spread is needed:

```suggestion
      style={{ ...style, ...imgStyle, borderRadius }}
```

Reply:
done